screen/Running/controllers/gameController.js

```diff
@@ -0,0 +1,65 @@
+class GameController {
+  constructor(
+    intervalId,
+    timeoutId,
+    locationObj,
+    locationRecord,
+    audioController,
+  ) {
+    this.intervalId = intervalId;
+    this.timeoutId = timeoutId;
+    this.locationObj = locationObj;
+    this.locationRecord = locationRecord;
+    this.audioController = audioController;
+  }
+
+  loadGameSound = () => {
+    this.audioController.loadAudio();
+  };
+
+  pauseGameStatus = () => {
+    clearInterval(this.intervalId.current);
+    this.locationObj.current?.remove();
+    this.intervalId.current = null;
+    this.audioController.stopAllSound();
+  };
+
+  recordUserHistory = (coords) => {
+    this.locationRecord.current = [
+      ...this.locationRecord.current,
+      {
+        latitude: coords.latitude,
+        longitude: coords.longitude,
+      },
+    ];
+  };
+
+  resetGameSetup = (resetTarget) => {
+    if (resetTarget === 'timer') {
+      clearInterval(this.intervalId.current);
+    }
+
+    if (resetTarget === 'timeout') {
+      clearTimeout(this.timeoutId.current);
+    }
+
+    this.audioController.resetAudio();
+    this.locationObj.current?.remove();
+  };
+
+  gameSoundControll = (hasEffectOn, hasMusicOn) => {
+    if (hasEffectOn) {
+      this.audioController.playSoundEffect();
+    }
+
+    if (hasMusicOn) {
+      this.audioController.playBackgroundMusic();
+    }
+  };
+
+  soundEffectVolumeControll = (volume) => {
```

Comment:
오타가 났습니다

```suggestion
  soundEffectVolumeControl = (volume) => {
```

위 리뷰와 마찬가지로 controlSoundEffectVolume도 좋을 것 같습니다.

Reply:
그렇네요. 순간 스펠링을 엘이 두개 인걸로 착각했었습니다. 다 수정하겠습니다.